src/gui/filedetails/sharemodel.cpp

```diff
@@ -24,7 +24,8 @@
 
 namespace {
 
-static const QString placeholderLinkShareId = QStringLiteral("__placeholderLinkShareId__");
+static const auto placeholderLinkShareId = QStringLiteral("__placeholderLinkShareId__");
```

Comment:
@claucambra Should these be `constexpr`?

Reply:
constexpr doesn't work with QStrings